test/e2e/specs/demo.test.js

```diff
@@ -17,47 +12,22 @@ const MOCK_VIMEO_RESPONSE = {
 	version: '1.0',
 };
 
+const couldNotBePreviewed = ( obj ) => {
```

Comment:
Lowercase `object` in JS isn't reserved, right? I really dislike variable abbreviations, but I'm of two minds here:

<dl>
<dt>pro:</dt><dd>Variable abbreviations are too retro for my taste; we have the bytes to spare 😉</dd>
<dt>con:</dt><dd>Seeing `Object` and `object` next to each other is confusing 😓</dd>
</dl>

Maybe we could just call this `embedObject` or something? 🤷‍♂️